storage_service/locations/models/package.py

```diff
@@ -1426,6 +1426,33 @@ def create_replicas(self):
         for replicator_loc in replicator_locs:
             self.replicate(replicator_loc)
 
+    def _replace_callback_placeholders(self, uri, body):
+        """
+        Replaces post store callback placeholders with values.
```

Comment:
Please use the imperative mood. See PEP257:

> The docstring is a phrase ending in a period. It prescribes the function or method's effect as a command ("Do this", "Return that"), not as a description; e.g. don't write "Returns the pathname ...".

I think that we're not doing it consistently though.

Reply:
Cosmetic, it should be:

        """Replace post store callback placeholders with values.

(without the line break)